### ISSUE TITLE
Colabfold MSA server connection error

### ISSUE THREAD

**Issue by alexlevylab** [sokrypton/ColabFold#481]
## Expected Behavior
I have been using v1.5 of localcolabfold and it has been working great. 

## Current Behavior
Ever since the recent issue with the server, I have not been able to connect to the MSA server:

## Steps to Reproduce (for bugs)
colabfold_batch --num-recycle 20 --recycle-early-stop-tolerance=0.5 --stop-at-score 90 --model-type alphafold2_multimer_v3 --rank multimer in out

## ColabFold Output (for bugs)
2023-07-31 19:44:45,545 Running colabfold 1.5.2 (f59147dbfb7b4be95178da1aa6b18b839a2ebee0)
2023-07-31 19:45:03,286 Running on GPU
2023-07-31 19:45:05,910 Found 4 citations for tools or databases
2023-07-31 19:45:05,914 Query 1/16: EFF01820_and_IMU38216 (length 1311)
2023-07-31 19:45:29,955 Error while fetching result from MSA server. Retrying... (1/5)
2023-07-31 19:45:29,957 Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Max retries exceeded with url: /ticket/msa (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0e889ac040>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2023-07-31 19:45:57,988 Error while fetching result from MSA server. Retrying... (1/5)
2023-07-31 19:45:57,990 Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Max retries exceeded with url: /ticket/msa (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0e889ac1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2023-07-31 19:46:28,016 Error while fetching result from MSA server. Retrying... (1/5)

FYI, I also tried this:

wget https://api.colabfold.com/queue

--2023-07-31 19:53:45--  https://api.colabfold.com/queue
Resolving api.colabfold.com (api.colabfold.com)... failed: Temporary failure in name resolution.
wget: unable to resolve host address ‘api.colabfold.com’

## Your Environment
I tried f59147dbfb7b4be95178da1aa6b18b839a2ebee0
and also the newest 63decf556dc2cabfb679917b86d5f3a526a481e7

I am running on Ubuntu

**Comment by milot-mirdita** [comment]
It is failing for you right now? It seems fine to me. Yesterday around ~11:00 CEST, we migrated back to the main server. There you might have experienced some issues, but right now it should work correctly.

**Comment by duhovka** [comment]
We have the same problem for the past few days...
Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Max retries exceeded with url: /ticket/msa (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdc88c24220>: Failed to establish a new connection: [Errno -2] Name or service not known'))

**Comment by milot-mirdita** [comment]
Is this error from today? This should not be happening anymore.

**Comment by milot-mirdita** [comment]
I think you two and two more users who reached out to me through other platforms are from the same institute. Apparently, your IT is aware of this issue, though AFAIK they haven't reached out to us directly yet.

For some reason your institute does not seem to resolve `api.colabfold.com` to its IP address. I reached out to our DNS/domain host and they said that the colabfold domain has recently had changes to its nameservers (I guess some internal thing) and that this might take a while to propagate. This should take up to 2 days. Ideally this issue will resolve itself automatically.

**Comment by alexlevylab** [comment]
Okay I see, good detective work!

I hope you're right and the nameserver changes simply need to propogate. I will pass along your message to our IT people and see if it helps them. Maybe there is something they can do in the meantime. 

In the meantime, I will keep the issue open for now while it's still an issue. 

Thanks,
Alex

**Comment by alexlevylab** [comment]
Looks like the issue was on our side. The IT people reset something to do with the DNS. It works now.

Thank you!

**Comment by chensena** [comment]
Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Read timed out.
2024-12-05 14:03:33,499 Timeout while fetching result from MSA server. Retrying...
The problem I've been encountering these days, how can I resolve it?

**Comment by milot-mirdita** [comment]
Please open and new issue. What is your IP?

**Comment by chensena** [comment]
hi! my IP is 183.230.2.125

**Comment by milot-mirdita** [comment]
I banned this IP range some time ago, as it was overloading the server. Please confirm that you will only send jobs from a single IP, before I unblock it again.

**Comment by chensena** [comment]
Dear Administrator,

Thank you for providing this service. I assure you that all tasks have been submitted from the same IP address. The reason for the overload might be that the sequences I submitted are relatively short, which allows the model building process to complete quickly, causing the system to become overloaded.

Best regards,

**Comment by milot-mirdita** [comment]
Thank you, I've unblocked the IP again.

**Comment by chensena** [comment]
Thanks

**Comment by chensena** [comment]
> Thank you, I've unblocked the IP again.

 Dear Administrator, I still cannot use the MSA service with IP 183.230.2.125 when i try again.iAre there any other issues?

**Comment by yanskysky** [comment]
> Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Read timed out. 2024-12-05 14:03:33,499 Timeout while fetching result from MSA server. Retrying... The problem I've been encountering these days, how can I resolve it?

Hi, I have met the same problem. What is the reason, and how to resolve it?

2025-03-20 08:02:41,666 Timeout while fetching result from MSA server. Retrying...                       
2025-03-20 08:03:22,085 Error while fetching result from MSA server. Retrying... (1/5)                   
2025-03-20 08:03:22,086 Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Read timed out.  
2025-03-20 08:03:35,869 Timeout while fetching result from MSA server. Retrying...                       
2025-03-20 08:04:39,705 Error while fetching result from MSA server. Retrying... (2/5)

**Comment by Tongye-Han** [comment]
@milot-mirdita 
Dear Administrator 

I also met this problem today  as follow, but  the colabfold and the server worked fine ten days ago.

2025-03-26 16:59:13,060 Error while fetching result from MSA server. Retrying... (1/5)
2025-03-26 16:59:13,063 Error: HTTPSConnectionPool(host='api.colabfold.com', port=443): Max retries exceeded with url: /ticket/msa (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x14cbb4166d40>: Failed to resolve 'api.colabfold.com' ([Errno -2] Name or service not known)"))
2025-03-26 16:59:36,090 Error while fetching result from MSA server. Retrying... (1/5)

Is this a result of recently performed maintenance on the main server, and if so, when will it be back to normal?

Thank you very much

**Comment by milot-mirdita** [comment]
Please make a new issue if the error persists

**Comment by Tongye-Han** [comment]
@milot-mirdita 

OK,  I have made a new issue which number is #697
